src/connectors/hierarchical-menu/__tests__/__snapshots__/connectHierarchicalMenu-test.js.snap

```diff
@@ -0,0 +1,93 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`connectHierarchicalMenu routing getWidgetSearchParameters should add the refinements according to the UI state provided 1`] = `
+SearchParameters {
+  "advancedSyntax": undefined,
```

Comment:
Would it be feasible to not have everything that is undefined and only what's relevant for the test?

Reply:
No unless the object was built in a different way (would require to change the JS Helper for that)